helm/terraform-job/templates/terraform-role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: "{{ .Values.serviceAccount.name }}-secret-admin-binding"
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
`helm.sh/hook-weight: "-5"`

needs to be created after Role and SA

Reply:
Made sure all the templates have the hook weight ordering with 49bfe48.